### ISSUE TITLE
Lua_error on index.lua

### ISSUE THREAD

**Issue by hvdbrand** [ntop/ntopng#330]
Hi,

I get the following error when connecting to my page.

HTTP/1.1 500 Internal server error Content-Type: text/html Connection: close Script "/usr/share/ntopng/scripts/lua/index.lua" returned an error:

/usr/share/ntopng/scripts/lua/modules/lua_utils.lua:14: bad argument #1 to 'pairs' (table expected, got nil)

I see a similar error in /var/log/messages/ :

Dec 20 07:50:59 server ntopng: [src/Lua.cpp:4478] WARNING: Script failure [/usr/share/ntopng/scripts/callbacks/second.lua][/usr/share/ntopng/scripts/lua/modules/lua_utils.lua:14: bad argument #1 to 'pairs' (table expected, got nil)]

Results from ntopng --version:

v.2.2.151211 [Professional Edition]
GIT rev:   2.2-stable:b96d564541333833293ce9537ebe598ffece5c3a:20151211
Pro rev:   r524
System Id: 8D8BA7399206AAF2
Built on:  CentOS Linux release 7.1.1503 (Core)

I use the ntop centos-stable repo at http://www.nmon.net/centos-stable

**Comment by simonemainardi** [comment]
I've just tested with 2.3.151229 - Professional Edition and it works just fine on CentOS Linux release 7.1.1503 (Core)

![image](https://cloud.githubusercontent.com/assets/3397663/12032465/e9a0f2f2-ae18-11e5-9a7d-176a1436b2a3.png)

The exception you have reported suggests me that interface statistics are not propagated to Lua. Is ntopng running with enough privileges?

Also the link you have enclosed is 404. Please see this: http://packages.ntop.org/centos/ and consider moving to the nightly build.

**Comment by hvdbrand** [comment]
Thanks for the reply.
The error I was experiencing in the logfile arose after an update via yum.
After stopping and restart the error in the logfile was gone.

I switched to the nightly build as suggested.
ntopng works fine on port 3000.
I noticed that the configuration files changed and had a time step of dec 11.
As a result SSH was disabled.
Changing this in the configuration file everything works as expected.
